x/msgfee/model_test.go

```diff
@@ -0,0 +1,83 @@
+package msgfee
+
+import (
+	"testing"
+
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/store"
+)
+
+func TestMsgFeeValidate(t *testing.T) {
+	cases := map[string]struct {
+		mf      MsgFee
+		wantErr error
+	}{
+		"all good": {
+			mf: MsgFee{
+				MsgPath: "foo/bar",
+				Fee:     coin.NewCoin(1, 2, "DOGE"),
+			},
+			wantErr: nil,
+		},
+		"empty path": {
+			mf: MsgFee{
+				MsgPath: "",
+				Fee:     coin.NewCoin(1, 2, "DOGE"),
+			},
+			wantErr: errors.ErrInvalidModel,
+		},
+		"zero value fee with a ticker": {
+			mf: MsgFee{
+				MsgPath: "foo/bar",
+				Fee:     coin.NewCoin(0, 0, "DOGE"),
+			},
+			wantErr: errors.ErrInvalidModel,
+		},
+		"zero value fee with no ticker": {
```

Comment:
Can you add one with non-zero values but invalid ticker symbol? like 2 chars long, or including `-`